src/bio/bio_device.c

```diff
@@ -672,3 +674,204 @@ bio_dev_list(struct bio_xs_context *xs_ctxt, d_list_t *dev_list, int *dev_cnt)
 
 	return rc;
 }
+
+int
+bio_set_led_state(struct bio_xs_context *xs_ctxt, char *dev_traddr,
```

Comment:
The "dev_traddr" could be removed from the parameters.

Reply:
The "xs_ctxt" is actually not used in this function. The only reason you pass it in is that you can put a D_ASSERT(is_init_xstream(xs_ctxt)) here.